deployment-examples/gcp/serverless/extension/README.md

```diff
@@ -0,0 +1,15 @@
+## Deploy as GCP Cloud Function
+
+1. Rename index.googleFunction.js to index.js
+
+2. For deployment to GCP Cloud Function, bundle the following files and folders into a .zip file and upload it in GCP console.
```

Comment:
when put in the package.json 
> "zip-serverless": "zip -r extension-function.zip src resources index.googleFunction.js package.json"

you might run it with:

```bash
npm run zip-serverless
```

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/2d265d2f0634972160936fa1fbb7a3a4f61b30bd

Reply:
Looks good... 

Can you add `notification-module.zip` and `extension-module.zip` to ignore files so we will not accidentally push them to the repo and docker container? (check .gitignore and .dockerignore files)

Reply:
I wonder should we will configure environment variables on the serverless docs ? otherwise it will fail, right, maybe it's better to put a link that the env variables should be set.